packages/websocket/package.json

```
@@ -1,10 +1,30 @@
 {
   "name": "@solid-primitives/websocket",
-  "version": "0.3.7",
-  "description": "Primitive to create a worker helper",
+  "version": "0.4.0",
+  "description": "Primitive to create a web socket connection",
   "author": "David Di Biase <dave.dibiase@gmail.com>",
+  "contributors": [
+    "Alex Lohr <alex.lohr@logmein.com>"
+  ],
   "license": "MIT",
   "homepage": "https://github.com/solidjs-community/solid-primitives/tree/main/packages/websocket",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/solidjs-community/solid-primitives.git"
+  },
+  "primitive": {
+    "name": "websocket",
+    "stage": 0,
+    "list": [
+      "makeWS",
+      "createWS",
+      "createMessageWS",
+      "makeReconnectingWS",
+      "createReconnectingWS",
+      "makeHeartbeatWS"
+    ],
+    "category": "network"
```

```suggestion
    "category": "Network"
```
not sure if our categories are case sensitive but better to stay consistent